### ISSUE TITLE
Attribute entity displayed as a number, but needle not working properly

### ISSUE THREAD

**Issue by Nikolay-Ch** [radimkeseg/lovelace-mini-gauge-card#3]
Ex, for configuration:
```yaml
- type: custom:mini-gauge-card
  entity: switch.socket.voltage
  title: Switch
  decimals: 2
  scale: 2
  severity:
    green: 200
    yellow: 180
    red: 0
  min: 150
  max: 300
```
![image](https://user-images.githubusercontent.com/24368057/154579944-79b7b289-f551-4b5e-8de3-364220186556.png)

**Comment by Nikolay-Ch** [comment]
Maybe you change line 253 of the file mini-gauge-card.js
from this:
```js
var mean = this._getEntityStateValue(hass.states[config.entity]);
```

to this:
```js
var mean = parseFloat(entityState).toFixed(config.decimals);
```

**Comment by radimkeseg** [comment]
fixed in latest commit
![image](https://user-images.githubusercontent.com/25863962/154651264-4d41ec49-628a-4e65-8069-dc5258a748e3.png)

will release soon

**Comment by radimkeseg** [comment]
fixed in 0.2.5.3

**Comment by Nikolay-Ch** [comment]
Ok, thank you! Maybe better delete line with defining **mean** variable (line 253)? Because it not used... ;)
